src/test/feature/Ranger/test_ranger.cpp

```diff
@@ -24,19 +24,34 @@
 
 #include "lib/command.h"
 #include "lib/gpfdist.h"
-#include "lib/sql_util.h"
 #include "lib/string_util.h"
 
 using std::vector;
 using std::string;
 using hawq::test::SQLUtility;
 using hawq::test::Command;
 
+TestHawqRanger::TestHawqRanger()
+{
+	initfile = hawq::test::stringFormat("Ranger/sql/init_file");
+	rangerHost = getRangerHost();
+}
+
+std::string& TestHawqRanger::getRangerHost()
```

Comment:
Better to get guc value by connecting to db and using sql from pg_settings, not hawq config. Using sql could avoid string find and hardcoded number length. And hawq config also connect to db and use sql to get the value.